src/frontend/utils/formatter.ts

```diff
@@ -13,9 +13,19 @@ export enum datoformatNorsk {
     DATO = 'DD.MM.ÅÅÅÅ',
 }
 
-export const formaterIsoDato = (dato: string | undefined, tilFormat: datoformat) => {
+export const formaterIsoDato = (
+    dato: string | undefined,
+    tilFormat: datoformat,
+    defaultString?: string
+) => {
     const momentDato = moment(dato);
-    return momentDato.isValid() ? momentDato.format(tilFormat) : dato ? dato : '';
+    return momentDato.isValid() && dato !== undefined
```

Comment:
I would use the simpler expression:
`return momentDato.isValid() && dato? momentDato.format(tilFormat) : dato || defaultString || '';`